experiments/tf_trainer/tf_gru_attention/run.py

```diff
@@ -0,0 +1,71 @@
+"""Experiments with Toxicity Dataset"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tf_trainer.common import tfrecord_input
+from tf_trainer.common import text_preprocessor
+from tf_trainer.common import types
+from tf_trainer.tf_gru_attention import model
+
+import nltk
+import tensorflow as tf
+
+from typing import Dict
+
+FLAGS = tf.app.flags.FLAGS
+
+tf.app.flags.DEFINE_string("train_path", "local_data/train.tfrecord",
+                           "Path to the training data TFRecord file.")
+tf.app.flags.DEFINE_string("validate_path", "local_data/validation.tfrecord",
+                           "Path to the validation data TFRecord file.")
+tf.app.flags.DEFINE_string("embeddings_path",
+                           "local_data/glove.6B/glove.6B.100d.txt",
+                           "Path to the embeddings file.")
+tf.app.flags.DEFINE_string("text_feature_name", "comment_text",
+                           "Feature name of the text feature.")
+tf.app.flags.DEFINE_string("model_dir", "/tmp/model_dir",
+                           "Directory for the Estimator's model directory.")
+
+# TODO: Missing fields are not handled properly yet.
+LABELS = {
+    "frac_neg": tf.float32,
+    #"frac_very_neg": tf.float32
+}  # type: Dict[str, tf.DType]
+
+
+def main(argv):
+  del argv  # unused
+
+  nltk.download("punkt")
```

Comment:
Do we still need this nltk download if we're doing tokenization in the graph? Also the import statement?

Reply:
Yes, still using py_func and relying on nltk to do tokenization. But this refactor allows easy switching.